templates/artist/includes/investor_profile_artist_list.html

```diff
@@ -11,6 +11,11 @@
             <div class="artist-list-image img-list">
                 {% include "object_preview.html" with object=artist img=artist.photo.img %}
             </div>
+            <div>
+                 <p class="green-color">
+                     ${{ artist.total_earned|floatformat:2|intcomma }}
+                 </p>
+             </div>
```

Comment:
We're already displaying this on the page a bit lower when `show_earnings` is `True`, which it seems to be for the private page.

That means this change for the private page is going to display the number twice, which seems wrong.

Are you trying to display the same information regardless of whether this is the public view or private view? If that's the case, we can just get rid of the `{% if show_earnings %}` block and always show the earnings.

Reply:
Ahh. I may have been confused here. My intention was to show the total earned on the public view and keep the private view the same.

Reply:
Okay just pushed a fix. I think the easiest thing is to just keep it how it was before